docs/lang/articles/performance_tuning/profiler.md

```diff
@@ -6,21 +6,18 @@ sidebar_position: 1
 
 ## Overview
 
-High-performance numerical computation is one of the key application domains of Taichi.
-We provide a suite of profilers to quantify the performance of Taichi programs, to help analyze where the bottleneck occurs,
-and thus facilitate users optimizing their code. These profilers collect both hardware and Taichi-related information
-and can also be used as performance debugging tools for developers.
+High-performance numerical computation is one of the key features of Taichi. To better measure the performance of Taichi programs, Taichi provides profiling tools that collect both hardware and Taichi-related information. They play an important role in the quantitative analysis of program performance, bottleneck identification, and optimization. They can also be used as debugging tools for developers.
```

Comment:
```suggestion
To facilitate code debugging and code optimization, Taichi provides a set of profiling tools, which collect hardware and Taichi-related information to quantify program performance and analyze where bottleneck occurs.
```
- Feature: selling point. By using "feature", you changed the intended meaning. 
- "To better measure" is not authentic English. 
-  I spent some time sorting out the causal relation in the sentence.